whois-api/src/main/java/net/ripe/db/whois/api/mail/EmailMessageInfo.java

```diff
@@ -2,5 +2,5 @@
 
 import java.util.List;
 
-public record EmailMessageInfo(List<String> emailAddresses, String messageId) {
+public record EmailMessageInfo(List<String> emailAddresses, String messageId, byte[] message) {
```

Comment:
Can you use a `MimeMessage` until you store it in the DAO as a `byte[]` ?

Reply:
I though that it was better to do this here because we are parsing the message.
and transforming it into bytes is part of a parsing procedure.
But I don't care to move it to the dao...however, the dao shouldn't do anything else rather that adding a record into a table...